### PR TITLE
Use hyphen instead of en dash

### PR DESCRIPTION
**Pull request by stevenjoezhang**
## PR Checklist
**Please check if your PR fulfills the following requirements:**

- [x] The commit message follows [our guidelines](https://github.com/theme-next/hexo-theme-next/blob/master/.github/CONTRIBUTING.md).
- [x] Tests for the changes was maked (for bug fixes / features).
   - [x] Muse | Mist have been tested.
   - [x] Pisces | Gemini have been tested.
- [ ] [Docs](https://github.com/theme-next/theme-next.org/tree/source/source/docs) in [NexT website](https://theme-next.org/docs/) have been added / updated (for features).

## PR Type
**What kind of change does this PR introduce?**

- [ ] Bugfix.
- [ ] Feature.
- [ ] Code style update (formatting, local variables).
- [ ] Refactoring (no functional changes, no api changes).
- [ ] Build related changes.
- [ ] CI related changes.
- [ ] Documentation content changes.
- [x] Other... Please describe:

## What is the current behavior?

Issue resolved: #899

## What is the new behavior?

- Screenshots with this changes: N/A
- Link to demo site with this changes: N/A

### How to use?
In NexT `_config.yml`:
```yml
...
```

## Does this PR introduce a breaking change?
- [ ] Yes.
- [x] No.

### CODE REVIEW

**Comment by 1v9** [comment]
Which one is longer?
HYPHEN (-) vs. EN DASH (–)

**Comment by stevenjoezhang** [comment]
https://cherysunzhang.com/2016/08/using-hyphen-and-dash-correctly/

File: LICENSE.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# <div align="center">«NexT» – Elegant and powerful theme for Hexo.</div>
```
Comment (1v9) [inline LICENSE.md]: Is this already en dash?

**Comment by Raincal** [comment]
只有 subtitle 那里需要改，其它地方不变

File: LICENSE.md (hunk 1 above)
Comment (1v9) [reply LICENSE.md]: Subtitle uses en dash now: `1v9's Blog – The magic ends here`, not hyphen.

**Comment by 1v9** [comment]
In fact I am a little dizzy, if I'm wrong please tell me.

**Comment by stevenjoezhang** [comment]
@Raincal 应保持统一。

File: LICENSE.md (hunk 1 above)
Comment (stevenjoezhang) [reply LICENSE.md]: No, it's hyphen. `"–"=="-"` returns false, you can have a try in JS console.

**Comment by Raincal** [comment]
目前用的都是 En dash(–)，只有 subtitle 那里需要改成 Hyphen(-)，你可能把两者搞混了

**Comment by stevenjoezhang** [comment]
@Raincal 根据vscode的搜索结果，
`—`(\u8212)出现了四次，在
`docs/ru/DATA-FILES.md`
`docs/ru/README.md`
`languages/ru.yml`
`languages/uk.yml`
猜测这是西里尔输入法打出的连字符。
`–`(\u8211)出现了四次，在
`docs/LICENSE.txt`
`layout/_partials/footer.swig`
`layout/index.swig`
`LICENSE.md`
中。这是中文输入法打出的破折号的一半。
因此：
> 目前用的都是 En dash(–)，

不，至少它们的unicode不同

> 只有 subtitle 那里需要改成 Hyphen(-)

为保持统一性和通用性，应全部替换为`-`(\u45)

> 你可能把两者搞混了

可能在交流上出现了一些误解。

**Comment by stevenjoezhang** [comment]
感觉是 #899 的描述有误。维基百科标题用的连字符是\u45。

**Comment by 1v9** [comment]
@stevenjoezhang Use hyphen instead of en dash

**Comment by stevenjoezhang** [comment]
@1v9 Seems we're talking about the same thing

**Comment by stevenjoezhang** [comment]
I know why. The title is wrong. Everything else is correct.

**Comment by Raincal** [comment]
> 为保持统一性和通用性，应全部替换为`-`(\u45)

使用场景不同，应该使用不同的符号，你可以再仔细看下你上面发的链接或者 wiki
https://zh.wikipedia.org/wiki/%E8%BF%9E%E6%8E%A5%E5%8F%B7#En_dash

**Comment by stevenjoezhang** [comment]
> 使用场景不同，应该使用不同的符号

目前的几个使用场景中连字符地位相同，统一使用\u45是合适的。

**Comment by 1v9** [comment]
我兹磁都用-，老实说 en dash 我都不会打 😂

**Comment by ivan-nginx** [comment]
Guys, there are 3 types of dash: small (hyphen), middle (for digits) and dash (long). I suggest to use **middle** dash for best vision for posts. Here we go, see:

1. In literature culture we like to use pretty-good hyphens for string combinations
2. If we want to calculate, we do it simply: 1+1=2; 1–1=0 (middle) 
3. Long dashes can be used for author replicas, – Say Ivan right now and drink a cup of tee.

**Comment by stevenjoezhang** [comment]
@ivan-nginx 

> In literature culture we like to use pretty-good hyphens for string combinations

Sites like Wikipedia are using `-`(\u45), aka small (hyphen), for string combinations
Maybe website is not kind of `literature culture` 😂

> If we want to calculate, we do it simply: 1+1=2; 1–1=0

Emm... not `simply`. When I press the key `-` on my keyboard, I got a `-`, not \u8211 . So, it should be 1+1=2; 1-1=0; not 1–1=0
`+` is \u43, and `-` is \u45. They are simple because ASCII code is simpler than Unicode.

**Comment by stevenjoezhang** [comment]
I found this:
#337
#341
interesting 😂

**Comment by 1v9** [comment]
@stevenjoezhang Enough, various expressions of this shit symbol... 😂

**Comment by 1v9** [comment]
Em Dash
&mdash; 
En Dash
&ndash; 
Hyphen
&dash;

**Comment by ivan-nginx** [comment]
Just add option for dash, no?

**Pull request merged**